awx/main/tasks.py

```diff
@@ -2790,42 +2790,19 @@ def emit(self, record):
             job_timeout=self.get_instance_timeout(self.instance),
             counter=self.event_ct
         )
+        handler._start_line = self.end_line
```

Comment:
this was kind of lazy, we could make the starting line a kwarg to the handler class